### ISSUE TITLE
Output is written twice to the console during training

### ISSUE THREAD

**Issue by pusankim** [tensorflow/models#5017]
OS Platform and Distribution 
WINDOWS 10

TensorFlow installed from
anaconda

TensorFlow version
tensorflow-gpu      1.9.0

Bazel version

CUDA/cuDNN version
9.0/6.0

GPU model and memory
GeForce GTX 1050 Ti, 4GB

Exact command to reproduce

INFO:tensorflow:global step 199972: loss = 0.0452 (0.290 sec/step)
INFO:tensorflow:global step 199972: loss = 0.0452 (0.290 sec/step)
INFO:tensorflow:global step 199973: loss = 4351578088673814511616.0000 (0.276 sec/step)
INFO:tensorflow:global step 199973: loss = 4351578088673814511616.0000 (0.276 sec/step)
INFO:tensorflow:global step 199974: loss = 4637446326820970430464.0000 (0.283 sec/step)
INFO:tensorflow:global step 199974: loss = 4637446326820970430464.0000 (0.283 sec/step)
INFO:tensorflow:global step 199975: loss = 0.0367 (0.284 sec/step)
INFO:tensorflow:global step 199975: loss = 0.0367 (0.284 sec/step)
INFO:tensorflow:global step 199976: loss = 0.0421 (0.286 sec/step)
INFO:tensorflow:global step 199976: loss = 0.0421 (0.286 sec/step)
INFO:tensorflow:global step 199977: loss = 0.0378 (0.285 sec/step)
INFO:tensorflow:global step 199977: loss = 0.0378 (0.285 sec/step)
INFO:tensorflow:global step 199978: loss = 3157714078379565645824.0000 (0.285 sec/step)
INFO:tensorflow:global step 199978: loss = 3157714078379565645824.0000 (0.285 sec/step)
INFO:tensorflow:global step 199979: loss = 0.0269 (0.272 sec/step)
INFO:tensorflow:global step 199979: loss = 0.0269 (0.272 sec/step)
INFO:tensorflow:global step 199980: loss = 1563645005548432130048.0000 (0.287 sec/step)
INFO:tensorflow:global step 199980: loss = 1563645005548432130048.0000 (0.287 sec/step)
INFO:tensorflow:global step 199981: loss = 0.0456 (0.300 sec/step)
INFO:tensorflow:global step 199981: loss = 0.0456 (0.300 sec/step)
INFO:tensorflow:global step 199982: loss = 0.0381 (0.283 sec/step)
INFO:tensorflow:global step 199982: loss = 0.0381 (0.283 sec/step)
INFO:tensorflow:global step 199983: loss = 0.0855 (0.281 sec/step)
INFO:tensorflow:global step 199983: loss = 0.0855 (0.281 sec/step)
INFO:tensorflow:global step 199984: loss = 0.0408 (0.280 sec/step)
INFO:tensorflow:global step 199984: loss = 0.0408 (0.280 sec/step)
INFO:tensorflow:global step 199985: loss = 1553882608931176448000.0000 (0.288 sec/step)
INFO:tensorflow:global step 199985: loss = 1553882608931176448000.0000 (0.288 sec/step)
INFO:tensorflow:global step 199986: loss = 1558102903994487668736.0000 (0.293 sec/step)
INFO:tensorflow:global step 199986: loss = 1558102903994487668736.0000 (0.293 sec/step)
INFO:tensorflow:global step 199987: loss = 0.0442 (0.280 sec/step)
INFO:tensorflow:global step 199987: loss = 0.0442 (0.280 sec/step)
INFO:tensorflow:global step 199988: loss = 0.0528 (0.276 sec/step)
INFO:tensorflow:global step 199988: loss = 0.0528 (0.276 sec/step)
INFO:tensorflow:global step 199989: loss = 3111071422938796261376.0000 (0.279 sec/step)
INFO:tensorflow:global step 199989: loss = 3111071422938796261376.0000 (0.279 sec/step)
INFO:tensorflow:global step 199990: loss = 0.0866 (0.276 sec/step)
INFO:tensorflow:global step 199990: loss = 0.0866 (0.276 sec/step)
INFO:tensorflow:global step 199991: loss = 0.0375 (0.274 sec/step)
INFO:tensorflow:global step 199991: loss = 0.0375 (0.274 sec/step)
INFO:tensorflow:global step 199992: loss = 3141269465815198990336.0000 (0.274 sec/step)
INFO:tensorflow:global step 199992: loss = 3141269465815198990336.0000 (0.274 sec/step)
INFO:tensorflow:global step 199993: loss = 0.0908 (0.272 sec/step)
INFO:tensorflow:global step 199993: loss = 0.0908 (0.272 sec/step)
INFO:tensorflow:global step 199994: loss = 0.0389 (0.274 sec/step)
INFO:tensorflow:global step 199994: loss = 0.0389 (0.274 sec/step)
INFO:tensorflow:global step 199995: loss = 0.0317 (0.282 sec/step)
INFO:tensorflow:global step 199995: loss = 0.0317 (0.282 sec/step)
INFO:tensorflow:global step 199996: loss = 0.0332 (0.283 sec/step)
INFO:tensorflow:global step 199996: loss = 0.0332 (0.283 sec/step)
INFO:tensorflow:global step 199997: loss = 1525433229347588669440.0000 (0.274 sec/step)
INFO:tensorflow:global step 199997: loss = 1525433229347588669440.0000 (0.274 sec/step)
INFO:tensorflow:global step 199998: loss = 4603309604595455492096.0000 (0.278 sec/step)
INFO:tensorflow:global step 199998: loss = 4603309604595455492096.0000 (0.278 sec/step)
INFO:tensorflow:global step 199999: loss = 0.0320 (0.281 sec/step)
INFO:tensorflow:global step 199999: loss = 0.0320 (0.281 sec/step)
INFO:tensorflow:global step 200000: loss = 3127472125406796054528.0000 (0.277 sec/step)
INFO:tensorflow:global step 200000: loss = 3127472125406796054528.0000 (0.277 sec/step)
INFO:tensorflow:Stopping Training.
INFO:tensorflow:Stopping Training.
INFO:tensorflow:Finished training! Saving model to disk.
INFO:tensorflow:Finished training! Saving model to disk.

Does the same notification pop up twice, which means that it has been trained twice?

From the beginning to the end of the training, INFO came up twice.

I would be grateful if someone could tell me how to fix it.

**Comment by apacha** [comment]
No, there seems to be a bug in the way, how the output is written to the console. It only trained once. I'm experiencing the same issue, that the output is written twice to the console.

**Comment by apacha** [comment]
And maybe you should consider renaming your issue to "Output is written twice to the console during training" instead of "I've trained with a tensor flow, and it's done, but there's a problem.", which is not a clear title.

**Comment by pusankim** [comment]
ok. thank you. Your advice is correct. I changed the title. Ah. So it's not training twice, it's not going to be a problem. Thank you for letting us know.

**Comment by apacha** [comment]
Interestingly this behavior only happens when I'm training with legacy slim training (new estimator-based training does not produce any output at all), and when training a faster-r-cnn based detector, not when training an SSD-based detector.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by apacha** [comment]
Well, @tensorflowbutler we would love to close this issue, if someone fixed it.

**Comment by codecolony** [comment]
Facing same issue.

**Comment by ravikyram** [comment]
@pusankim 

Is this still an issue?
Please, close this thread if your issue was resolved.Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
